### PR TITLE
Us 10318 language toggle

### PR DESCRIPTION
**Pull request by AkhilaHMRC**
Implemented Language toggle component
Added the i18next for language translation at front end (json for the translated frontend content will be implemented in US-10319)
Added code to update the Locale for pega content

### CODE REVIEW

File: src/applications/ChildBenefitsClaim/StartPage.tsx [hunk 1]
```diff
@@ -1,19 +1,21 @@
-import React from 'react'
-import Button from '../../components/BaseComponents/Button/Button'
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Button from '../../components/BaseComponents/Button/Button';
 
 const StartPage: React.FC<{onStart: React.MouseEventHandler, onBack: any}> = ({onStart, onBack}) => {
+  const { t } = useTranslation();
```
Comment (monalisasarkar123) [inline src/applications/ChildBenefitsClaim/StartPage.tsx:6]: Is there any particular reason for using {t} as constant name, if not please change it.

File: src/applications/ChildBenefitsClaim/UserPortal.tsx [hunk 2]
```diff
@@ -1,28 +1,28 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import Button from '../../components/BaseComponents/Button/Button';
 
 export default function UserPortal(props){
 
   const { beginClaim, children } = props;
+  const { t } = useTranslation();
```
Comment (monalisasarkar123) [inline src/applications/ChildBenefitsClaim/UserPortal.tsx:8]: same here

File: tsconfig.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "./dist/",
     "experimentalDecorators": true,
-    "strict": true,
+    "strict": false,
```
Comment (monalisasarkar123) [inline tsconfig.json:5]: have we changed this to false?

File: src/applications/ChildBenefitsClaim/StartPage.tsx [hunk 4]
```diff
@@ -75,8 +77,14 @@ const StartPage: React.FC<{onStart: React.MouseEventHandler, onBack: any}> = ({o
           Start now
         </Button>
         <p className='govuk-body'>
-          If you have any questions about Child Benefit, <a
-            href='https://www.gov.uk/government/organisations/hm-revenue-customs/contact/child-benefit' className='govuk-link'>contact the Child Benefit helpline</a>.
+          If you have any questions about Child Benefit,{' '}
```
Comment (monalisasarkar123) [inline src/applications/ChildBenefitsClaim/StartPage.tsx:80]: {' '}?

**Review by monalisasarkar123 — CHANGES_REQUESTED** [review]

File: src/applications/ChildBenefitsClaim/StartPage.tsx (hunk 1 above)
Comment (AkhilaHMRC) [reply src/applications/ChildBenefitsClaim/StartPage.tsx:6]: t is short for translate

File: src/applications/ChildBenefitsClaim/StartPage.tsx (hunk 4 above)
Comment (AkhilaHMRC) [reply src/applications/ChildBenefitsClaim/StartPage.tsx:80]: this would be fixed in another translation story

**Review by monalisasarkar123 — APPROVED** [review]

**Pull request merged**